template/demo_name.py

```diff
@@ -0,0 +1,21 @@
+# Copyright [yyyy] [name of copyright owner]
+# 
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+# 
+#     http://www.apache.org/licenses/LICENSE-2.0
+# 
+#     Unless required by applicable law or agreed to in writing, software
+#     distributed under the License is distributed on an "AS IS" BASIS,
+#     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#     See the License for the specific language governing permissions and
+#     limitations under the License.
+
+
+"""
+Things to do:
+ - Fill in [yyyy] and [name of copyright owner] in the license
+ - Add demo code below
+ - Format code so that it conforms with PEP 8
+"""
```

Comment:
In test_demo_file.py you also have the reminder "Please name this file `test_<demo_name>.py`", might as well have one here too